core_acc_modules/gene_relationships.py

```diff
@@ -253,3 +253,94 @@ def get_relationship_in_expression_space(
     # total_counts = expression_dist_counts.groupby("offset")["total"].sum()[1]
     # expression_dist_counts["proportion"] = expression_dist_counts["total"]/total_counts
     return expression_dist_counts
+
+
+def find_related_acc_genes(
+    corr_df,
+    genes_to_consider,
+    gene_mapping_df,
+    num_top_genes,
+    operon_df=None,
+
+):
+
+    """For each gene in `genes_to_consider`, is the highest correlated, 2nd-highest correlated, 3rd
+    highest correlated gene, etc core or accessory? This function
+    finds the accessory genes that are highest correlated,
+    next highest correlated, third highest correlated, etc.
+    This function returns a dataframe with `genes_to_consider` as the index and the list
+    of accessory genes that they are most correlated with.
+
+    Arguments:
+    ----------
+    corr_df: pandas df
+        Dataframe containing the correlation matrix. The matrix is a gene x gene
```

Comment:
Where is this correlation matrix made?

Reply:
Good point. I will add this to the description